src/pages/methods/WorkflowList.tsx

```diff
@@ -33,7 +33,7 @@ const defaultTabKey: TabKey = tabKeys[0];
 
 /**
  * Represents a list of method definitions grouped into two
```

Comment:
Should the name of this directory change from `/methods` to `/workflows`?

Reply:
See https://github.com/DataBiosphere/terra-ui/pull/5204#issuecomment-2569812821

Reply:
https://github.com/DataBiosphere/terra-ui/pull/5210